app/__init__.py

```diff
@@ -4,8 +4,10 @@
 from sqlalchemy import create_engine, Column, Integer, String, func
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from flask_babel import Babel
 
 app = Flask(__name__)

```

Comment:
No, they can't be in the module.
You should have made a package ("books" folder) for the whole books blueprint.
Inside that blueprint you can decorate your views like you do now.
App will import books and initialize it inside create_app.